src/clib/lib/enkf/gen_kw_config.cpp

```diff
@@ -102,6 +104,14 @@ void gen_kw_config_set_parameter_file(gen_kw_config_type *config,
         config_content_type *content =
             config_parse(parser, parameter_file, "--", NULL, NULL, NULL,
                          CONFIG_UNRECOGNIZED_ADD, false);
+        if (!content->valid) {
+            logger->warning(
```

Comment:
This shows up as multiple logging statements in azure, which is difficult to deal with. Could we make it into one statement?